### ISSUE TITLE
Poor Exception raised when model not set

### ISSUE THREAD

**Issue by ishaan-jaff** [BerriAI/litellm#276]
I got this stack trace when I didn't set a LLM for completion()
example:

response = completion(
            deployment_id="chatgpt-v-2",
            messages=messages,
        )
        

was really hard to decipher what the problem was:

```
Traceback (most recent call last):
  File "/Users/ishaanjaffer/Github/litellm/litellm/utils.py", line 524, in wrapper
    result = original_function(*args, **kwargs)
  File "/Users/ishaanjaffer/Github/litellm/litellm/timeout.py", line 42, in wrapper
    result = future.result(timeout=local_timeout_duration)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/concurrent/futures/_base.py", line 445, in result
    return self.__get_result()
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/concurrent/futures/_base.py", line 390, in __get_result
    raise self._exception
  File "/Users/ishaanjaffer/Github/litellm/litellm/timeout.py", line 33, in async_func
    return func(*args, **kwargs)
TypeError: completion() missing 1 required positional argument: 'model'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ishaanjaffer/Github/litellm/litellm/tests/test_completion.py", line 342, in test_completion_azure_deployment_id
    response = completion(
  File "/Users/ishaanjaffer/Github/litellm/litellm/utils.py", line 552, in wrapper
    threading.Thread(target=logging_obj.failure_handler, args=(e, traceback_exception, start_time, end_time)).start()
AttributeError: 'NoneType' object has no attribute 'failure_handler'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ishaanjaffer/Github/litellm/litellm/tests/test_completion.py", line 350, in <module>
    test_completion_azure_deployment_id()
  File "/Users/ishaanjaffer/Github/litellm/litellm/tests/test_completion.py", line 349, in test_completion_azure_deployment_id
    pytest.fail(f"Error occurred: {e}")
  File "/Users/ishaanjaffer/Library/Python/3.9/lib/python/site-packages/_pytest/outcomes.py", line 198, in fail
    raise Failed(msg=reason, pytrace=pytrace)
Failed: Error occurred: 'NoneType' object has no attribute 'failure_handler'
```

**Comment by ghost** [comment]
Addressing this now - a9cab12a474ac1ad029a88982639a28da7ea3dd2

**Comment by ghost** [comment]
updated to fix a bug - 263704e942cb4b306383574b0779d5b4ab82ea36
